.github/workflows/publish.yml

```diff
@@ -28,19 +28,12 @@ jobs:
           java-version: "17"
 
       - name: Checkout source code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: Build Docker images
         run: |
           ./gradlew docker
 
-      - name: Run Docker benchmark security tests
-        run: |
-          cd ..
-          git clone https://github.com/docker/docker-bench-security.git
-          cd docker-bench-security
-          sh docker-bench-security.sh -c container_images -i abd_vro -e check_4_5
-
```

Comment:
`Run cis-benchmark tests` removed from StarterKit: https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/eb6d84fd3cdc02ab2ac97875af086598b53514d3?diff=split#diff-7829468e86c1cc5d5133195b5cb48e1ff6c75e3e9203777f6b2e379d9e4882b3L29